translations/sv.json

```diff
@@ -25,7 +25,6 @@
 	"WNW": "VNV",
 	"NW": "NV",
 	"NNW": "NNV",
-	"FEELS": "Känns som", 
```

Comment:
Just curious: why is this translation removed?

Reply:
Because its also in line 35 :-)

Reply:
Ah, yes. GitHub didn't show that! :)